.github/workflows/release.yaml

```diff
@@ -41,4 +41,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          gh release upload ${GITHUB_REF_NAME} build/vmnet-helper-${GITHUB_REF_NAME}-*.tar.gz
+          gh release upload ${GITHUB_REF_NAME} build/vmnet-helper-*.tar.gz
```

Comment:
why not upload two files one with verison number and one with without so users relying version number can still be not broken